### ISSUE TITLE
Fix broken link in docs

### ISSUE THREAD

**Issue by rcrichton** [jembi/openhim-core-js#635]
The tutorial page's link to the tutorial docs is broken http://openhim.readthedocs.org/en/latest/tutorial/1-getting-started.html

**Comment by hnnesv** [comment]
It might be a good idea to host the tutorial code in github repos

**Comment by rcrichton** [comment]
Definitely. The source can be downloaded directly as a .zip from there.

**Comment by magp18** [comment]
The tutorial services link is still broken for me, or did I miss something?

**Comment by BMartinos** [comment]
Hi @magp18 The documentation for the OpenHIM has been moved away from Read The Docs and into its own website. Please find all updated documentation here
http://openhim.org/

**Comment by magp18** [comment]
Oh I see, thank you very much for the prompt reply!

**Comment by BMartinos** [comment]
No problem :)
